data/ocua_18-19/session2/week3_game1.json

```diff
@@ -2069,7 +2075,7 @@
         "Travis Davidson",
         "Nick Theriault",
         "Patrick Kenzie",
-        "Ren\u00e9e Gauvin",
+        "Renée Gauvin",
```

Comment:
Looks like my backfill changed these characters back. I'll have to figure out how to handle that right.